### ISSUE TITLE
Edgeshark integration.

### ISSUE THREAD

**Issue by DanPartelly** [ipspace/netlab#1441]
### Discussed in https://github.com/ipspace/netlab/discussions/1423

<div type='discussions-op-text'>

<sup>Originally posted by **DanPartelly** October 23, 2024</sup>
Do we want Edgeshark integration in external tools , like there is for graphite ? For full functionality the user would have to install the wireshark capture plugins on their own. </div>

**Comment by DanPartelly** [comment]
Turns out adding Edgeshark as external tool support  can be done with a template only. With a caveat: edgeshark should only be launched once, even in the case of a multilab. It auto-discovers all networking on the Linux host. 

I looked to see if multilab instance ids follow a strict ordering, to launch edgeshark only for the first lab instance within a multilab. But it seems that the IDs can be arbitrary.   Any ideas how this should be handled to fit well ?
